multiworld/core/wrapper_env.py

```diff
@@ -15,3 +17,93 @@ def __getattr__(self, attrname):
             return None
         return getattr(self._wrapped_env, attrname)
 
+class NormalizedBoxEnv(ProxyEnv, Serializable):
+    """
+    Normalize action to in [-1, 1].
+
+    Optionally normalize observations and scale reward.
+    """
+    def __init__(
+            self,
+            env,
+            reward_scale=1.,
+            obs_means=None,
+            obs_stds=None,
+            obs_to_normalize_keys=['observation'],
+    ):
+        # self._wrapped_env needs to be called first because
+        # Serializable.quick_init calls getattr, on this class. And the
+        # implementation of getattr (see below) calls self._wrapped_env.
+        # Without setting this first, the call to self._wrapped_env would call
+        # getattr again (since it's not set yet) and therefore loop forever.
+        # Or else serialization gets delegated to the wrapped_env. Serialize
+        # this env separately from the wrapped_env.
+        self._wrapped_env = env
+        self._serializable_initialized = False
```

Comment:
FYI, I think we don't need these two lines any more

Reply:
Ok sounds good, I will delete them then!